src/element.cpp

```diff
@@ -649,6 +659,14 @@ namespace libsemigroups {
         }
       }
     }
+    for (size_t u = 0; u < n; ++u) {
+      if (!std::is_sorted(this->_vector.at(u).cbegin(),
+                          this->_vector.at(u).cend())) {
+        LIBSEMIGROUPS_EXCEPTION("the adjacencies of vertex ",
+                                detail::to_string(u),
```

Comment:
This needs to be `detail::to_string(u).c_str()` to work when `fmt` is not enabled.

Reply:
```suggestion
                                detail::to_string(u).c_str(),
```